### PR TITLE
Clown Spacesuit

### PR DESCRIPTION
**Pull request by Shadowmech88**
Adds the clown space helmet and clown spacesuit.
clown_suit
The suit squeaks like the clown shoes do when you walk.
Credit to @Blithering for the sprites.

The suit and helmet are currently only obtainable via adminbus.

### CODE REVIEW

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 1]
```diff
@@ -52,4 +54,35 @@
...
+/obj/item/clothing/head/helmet/space/clown
+	name = "clown helmet"
+	desc = "The large grinning clown face on the front of the helmet is equal parts funny and creepy."
+	icon_state = "clown-eva-helmet"
+	item_state = "clown-eva-helmet"
+	species_restricted = list("exclude","Vox")
+
+/obj/item/clothing/suit/space/clown
+	name = "clown spacesuit"
+	desc = "Many clowns tragically drowned in space before before the duck floaty was added to the suit's design."
```
Comment (ghost) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: On this alone, :+1:

**Comment by CptWad** [comment]
at least its not hippes clown suit which looks ass

**Comment by Exxion** [comment]
:+1: AS LONG AS it remains admin-only

**Comment by SonixApache** [comment]
Are those clown magboots? If so, bus only. :+1:

File: code/modules/clothing/spacesuits/miscellaneous.dm (hunk 1 above)
Comment (NigglyWiggly) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: That's wonderful

Comment (ihadtoregisterforthis) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: ˃`before before`

**Comment by Shadowmech88** [comment]
@SonixApache 
The clown shoes are a part of the suit, they confer no magboot protection.

**Comment by Ephx1** [comment]
I wouldn't mind this as a Traitor Clown space suit. Just the normal red softsuit but looking like this.

**Comment by Exxion** [comment]
That would allow for some impressively fast valid collection

**Comment by Ephx1** [comment]
Same with balloons, or any real tater item?

**Comment by Exxion** [comment]
Balloons are specifically for telling everyone you're a traitor because you're so badass you can handle everyone knowing
The other items can be concealed
Red space suits can be acquired by non-antags so they aren't as conspicuous

**Comment by Ephx1** [comment]
> Balloons are specifically for telling everyone you're a traitor because you're so badass you can handle everyone knowing

Same with this clown space suit otherwise just get a normal one you honkless scrub.

**Comment by Exxion** [comment]
I was under the impression it was for traveling in space

**Comment by Ephx1** [comment]
That doesn't really change anything, you can get a normal suit in a few moments with almost zero effort.
It would be a nice flair to clowns scooting in space with style.

**Comment by Exxion** [comment]
Wait a second
Why don't we just make it constructible out of a normal space suit and clown gear or something?

**Comment by Ephx1** [comment]
That works too I guess. But then we would need a Mime Space suit that's invisible and only they can use.

**Comment by clusterfack** [comment]
Confrictu

**Comment by gbasood** [comment]
Fix conflicts
Also fix "before before"
Also, not for this PR, but it'd be cool if clowns got this as an uplink option.

**Comment by Blithering** [comment]
Thanks for making this. As to the balance issue, why not put one on the clown shuttle? Makes sense for it to be there, and anyone getting to it will already have a space suit.

**Comment by Shadowmech88** [comment]
This is ready for merge, I don't plan on doing more than adding it as an item.

**Pull request merged**